com.ibm.jbatch.tck/src/main/java/com/ibm/jbatch/tck/tests/jslxml/ParallelExecutionTests.java

```diff
@@ -598,6 +599,72 @@ public void testPartitionedMapperOverrideTrueSamePartitionNumOnRestart() throws
         }
     }
     
+    @TCKTest(
+    	versions = {"1.1.WORKING"},
+    	assertions = {"Each partition of a partitioned step has it's own unique Persistent User Data."},
+    	specRefs = {
+    		@SpecRef(
+    				version = "1.0RevA", section = "9.4.1.1",
+    				citations = "For a partitioned step, there is one StepContext for the parent step/thread; there is a distinct StepContext for each sub-thread "
+    						  + "and each StepContext has its own distinct persistent user data for each sub-thread.",
+    				notes = "See 2. StepContext"
+    			),
+    		@SpecRef(
+    			version = "1.0", section = "10.9.2",
+    			citations = "The setPersistentUserData method stores a persistent data object into the current step. [...] This data is saved as part of a step's "
+    					  + "checkpoint. [...] It is available upon restart.",
+    			notes = "APIRef for StepContext"
+    		)
+    	},
+    	apiRefs = {	@APIRef(className="javax.batch.runtime.context.StepContext", methodNames={"setPersistentUserData", "getPersistentUserData"}) },
+    	issueRefs = {"https://github.com/WASdev/standards.jsr352.tck/issues/18"},
+    	strategy = "See comments in the code for this test"
```

Comment:
I think the last discussion we had about the annotations was that we should make them smaller, and that the test strategy aka implementation details don't need to show up in the TCK test coverage report